week-4/E-arrays-of-objects/exercise-3.js

```diff
@@ -60,7 +60,9 @@ var restaurantFinderApplication = {
     applicationVersion: "1.0",
     restaurants: restaurants,
     findAvailableRestaurants: function(numberOfPeople) {
-        // Complete here
+        if(numberOfPeople < this.restaurants.totalSeats){
+            return this.restaurants.name;
+        }// Complete here
     },
```

Comment:
Not quite right. `this.restaurants` is an array so it won't have a `totalSeats` or `name` property.